mitiq/pec/representations/learn_noise_parameters.py

```diff
@@ -0,0 +1,125 @@
+# Copyright (C) 2022 Unitary Fund
```

Comment:
The name of this file `learn_noise_parameters.py` looks more like a function name than a file name.
I would suggest a simpler and shorter name for the file. 
E.g. `learning.py`

Reply:
good point- changed the filename to `learning.py`